### PR TITLE
Pull request is the first of many for building out this componentFeature/child/winner

### PR DESCRIPTION
**Pull request by just-kato**
# Pull Request Template

After the "scores" screen I installed a route that will showcase the winner. I created a "Winner" folder that houses the Winner component. Within that folder I created 3 files: index.js, RenderWinnerPage.js, WinnerPageContainer.js I did not yet add content to these files. I wanted to get the files merged before continuing.

## Trello Card

https://trello.com/c/C0WQBlMt/683-child-winner-route-and-component-creation

## Commits Checklist

- [x ] commits have clear title names
- [x ] commit has descriptive message of what has changed
- [x ] more small commits than less big commits

## Code Checklist

- [x ] create protected route
- [ x] create Winner folder under components/pages/Winner
- [ x] create index.js in Winner folder
- [ x] create RenderWinnerPage.js in Winner folder
- [ x] create WinnerPageContainer.js in Winner folder
- [ x] import Winner component into index.js

### CODE REVIEW

**Review by artofmayhem — CHANGES_REQUESTED** [review]
Did you make a construction page? You can take it right from the one I did on ya play/trophy-room if not. They're all going to be identical so at least something renders saying under construction.

**Comment by just-kato** [comment]
> Did you make a construction page? You can take it right from the one I did on ya play/trophy-room if not. They're all going to be identical so at least something renders saying under construction.

File: src/components/pages/WinnerPage/RenderWinnerPage.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    <>
+      <Header displayMenu={true} />
+      {/* REMOVE ONCE DESIGN WORK BEGINS - START */}
+      <div className={'under-construction'}>
+        <div className={'rectangle-126'} style={{ margin: '3rem' }}>
+          <h1>Coming Soon</h1>
+        </div>
+        <div className={'under-construction-content'}>
+          <h1>Under Construction!</h1>
+          <h1>Winner Page UI Coming Soon!</h1>
```
Comment (MichaelMcFaddenjr) [inline src/components/pages/WinnerPage/RenderWinnerPage.js:24]: Thank you for identifying what the page will be upon completion

**Review by MichaelMcFaddenjr — DISMISSED** [review]
Nice work.

**Review by rhiannonstanford — DISMISSED** [review]
Nice work filling in the gap for Winner page.  This will aid the route refactor.

**Review by artofmayhem — DISMISSED** [review]

**Review by rhiannonstanford — APPROVED** [review]
Looks good with changes from resolved merge conflict.

**Review by artofmayhem — APPROVED** [review]
thank you for resolving merge conflicts @ashtilawat23! Rock star status

**Pull request merged**